test.js

```diff
@@ -0,0 +1,79 @@
+const buttons=document.querySelectorAll('button');

```

Comment:
✨P1: 현재 reset 버튼이 적용되지 않는 이유는 자바스크립트에서 잘못된 선택자를 선택했기 때문입니다. 이 코드 때문에, 초기화 버튼을 역시 buttons에 포함되어 클릭하면 play 함수를 실행하게 됩니다. 위의 코드를 수정해보겠습니다.

```suggestion
const rockBtn = document.querySelector(".rock");
const paperBtn = document.querySelector(".paper");
const scissorBtn = document.querySelector(".scissor");
const buttons = [rockBtn, paperBtn, scissorBtn];
```
이렇게 작성하면 초기화 버튼은 선택하지 않고, 가위/바위/보 버튼만 선택할 수 있습니다. 또한 이를 buttons라는 배열에 담으면 기존에서 작성했던 forEach 메서드를 오류없이 사용할 수 있습니다.